packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -169,13 +178,11 @@ Widget _wrapWithBackground({
     child: result,
   );
 
-  if (backgroundColor.alpha == 0xFF) {
-    return childWithBackground;
-  }
-
   return ClipRect(
     child: BackdropFilter(
-      filter: ImageFilter.blur(sigmaX: 10.0, sigmaY: 10.0),
+      filter: backgroundColor.alpha == 0xFF || !isContentScrolledUnder
+        ? ImageFilter.blur()
```

Comment:
I wrote that to show it works but we'd need an `ImageFilter.noOp()` or similar instead.

Reply:
It looks like in the examples when it's supposed to be opaque there's still a blur. Is this why?

Reply:
By examples you mean the new screen recordings I posted? 

In that case it's normal to see blur everywhere since it did not make an example with a fully opaque background. 

I can make one if needed.

Reply:
If you wouldn't mind.

Reply:
I posted another one in the main comment section. About the `ImageFilter.blur()`, it works as expected as it has no visual effect. So having an `ImageFilter.noOp()` is to make it explicit we want a no op here.

Regarding the new demos that I pushed, is the visual result matching what you were expected as behavior?

Reply:
Yes it does, thank you. It looks great.

I think this is more than sufficient for this PR. I'd wonder if a lerp on the Blur would be the smoothest way to do it in the future. Just thinking out loud.

Reply:
you should add an `enabled` flag to BackdropFilter widget like we have for ImageFilter:

https://api.flutter.dev/flutter/widgets/ImageFiltered/ImageFiltered.html

There is no such thing as a no-op filter, this will still compute a blur which can be expensive.